test/server/clientlib.py

```diff
@@ -38,6 +38,12 @@ def login(self):
         resp.raise_for_status()
         return resp
 
+    def fxa_login(self, account_id):
+        device_id = get_device_id(account_id)
+        if device_id:
+          account_info = get_account_cookie_detail(account_id)
+          self.session.cookies.update(account_info)
```

Comment:
A couple issues here:

- `fxa_login()` doesn't make it clear what's being logged in. Are you logging the session in? Or is the device being associated with the accountId, as happens when you sign in with the add-on installed?
- There should a method to log the device into an account. (This method would have to call out to pglib, then call `self.login()`)
- For separate reasons we want a session to be able to get an accountId, without getting a deviceId. I don't think this method should do that.
- Testing device_id, but not testing that it's the same as `self.device_id` isn't good.
- You have two spaces instead of four ;)

Reply:
> A couple issues here:
> 
>     * `fxa_login()` doesn't make it clear what's being logged in. Are you logging the session in? Or is the device being associated with the accountId, as happens when you sign in with the add-on installed?

Updated to move this method in ScreenshotsBase class to simulate browser only session with user logged in with accountId.

>     * There should a method to log the device into an account. (This method would have to call out to pglib, then call `self.login()`)
> 
My understanding ScreenshotsClient class login method logs the device into an account by calling attach_device which can be used to setup an account. 

>     * For separate reasons we want a session to be able to get an accountId, without getting a deviceId. I don't think this method should do that.
> 
Using ScreenshotsBase class from clientlib the above scenario should suffice.

>     * Testing device_id, but not testing that it's the same as `self.device_id` isn't good.
> 
Updated tests with check
>     * You have two spaces instead of four ;)
done